indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -390,6 +390,20 @@ class RHRegistrationsConfigBadges(RHRegistrationsActionBase):
 
     ALLOW_LOCKED = True
 
+    format_map_portrait = {
+        'A0': (84.1, 118.9),
+        'A1': (59.4, 84.1),
+        'A2': (42.0, 59.4),
+        'A3': (29.7, 42.0),
+        'A4': (21.0, 29.7),
+        'A5': (14.8, 21.0),
+        'A6': (10.5, 14.8),
+        'A7': (7.4, 10.5),
+        'A8': (5.2, 7.4),
+    }
+
+    format_map_landscape = {frm: (frm_size[1], frm_size[0]) for frm, frm_size in format_map_portrait.iteritems()}
```

Comment:
What about this?

    format_map_landscape = {name: (h, w) for name, (w, h) in format_map_portrait.iteritems()}